regparser/grammar/atomic.py

```diff
@@ -60,7 +60,7 @@
 
 section = Word(string.digits).setResultsName("section")
 
-appendix = Regex(r"[A-Z]+[0-9]*\b").setResultsName("appendix")
+appendix = Regex(r"[A-Z]+-?[0-9]*\b").setResultsName("appendix")
```

Comment:
I don't think this should be needed anymore — the dash in Appendix MS is taken care of in the change in regparser/grammar/interpretation_headers.py now.